ogc_ws/src/despatcher/src/air_despatcher.py

```diff
@@ -35,6 +35,7 @@
 from mavros_msgs.srv import WaypointSetCurrent
 from std_msgs.msg import String
 from statustext.msg import YonahStatusText
+from despatcher.msg import Message
```

Comment:
Change Message to LinkMessage

Edit: Remember to change it across this file too

Reply:
I changed the relevant air_despatcher to LinkMessage but I realised on Rumesh's commit [https://github.com/yonahbox/Yonah_ROS_packages/commit/37a65ac98efc7552e36542e4c59d8bc7af672065](url) he has added additional things in gnd_despatcher and air_despatcher which he has changed to LinkMessage as well. So I think it will be easier to wait for his PR instead of changing it on my branch